core/domain/email_manager.py

```diff
@@ -328,6 +328,28 @@ def _send_bulk_mail_in_transaction(instance_id=None):
         _send_bulk_mail_in_transaction, instance_id)
 
 
+def send_job_failure_email(job_id):
```

Comment:
Add a unit test for this in email_manager_test.py.

Reply:
Done